docs/src/modules/components/AppDrawer.js

```diff
@@ -100,7 +100,7 @@ function AppDrawer(props, context) {
       <div className={classes.toolbarIe11}>
         <div className={classes.toolbar}>
           <Link className={classes.title} href="/" onClick={onClose}>
-            <Typography variant="title" color="inherit">
+            <Typography variant="headline6" color="inherit">
```

Comment:
For the sake of brevity, I'm wondering if we shouldn't be using "h" instead of "headline". It's something people are already used to. Here it would be `variant="h6"`. It's an open suggestion, I have no convinctions.

Reply:
Me neither. I was looking at the material-web-components implementation and they use `headlineX`. Using `hX` would match the spec.

Reply:
@mui-org/core-contributors What do you think?

Reply:
I think that I have got my answer 😆 
![capture d ecran 2018-09-23 a 15 48 07](https://user-images.githubusercontent.com/3165635/45928695-1c691780-bf48-11e8-8ba5-29ccc41a5be3.png)

Reply:
I vote for `h6`.

Reply:
I kind of like that we abstract away the actual html markup. It looks more consistent if we write out the meaning for every variant and not just a select few. But in this instance I don't really care how we go about naming these. I'm interested in what twitter has to say about it though. Did you already tweet this out @oliviertassinari?